draft-ietf-oauth-jwt-introspection-response.xml

```diff
@@ -190,22 +190,22 @@
 
 	<section anchor="jwt_request" title="Requesting a JWT Response">
     
-	<t>A resource server requests to receive a JWT introspection response by
-        including an Accept header with content type "application/jwt" in the
-        introspection request.</t>
+	<t>A resource server requests a JWT introspection response by including an
+    <spanx style="verb">Accept</spanx> HTTP header "application/token-introspection+jwt"
+    in the introspection request.</t>
         
     <t>Authentication at the token introspection endpoint can utilize client authentication
```

Comment:
```suggestion
   <t>The AS SHOULD authenticate the caller of the token introspection endpoint. Authentication at can utilize client authentication methods or a separate access token issued to the resource server. </t>
```

Reply:
@tlodderstedt the suggestion doesn't read well, the second sentense i guess has an extra `at`?

Reply:
yep, you are right.